contrib/debian/rules

```diff
@@ -18,7 +18,7 @@ override_dh_auto_clean:
 # Yea, autogen should be run on the source archive, but I like doing git archive
 override_dh_auto_configure:
 	./autogen.sh
-	./configure
+	./configure --without-miniupnpc --with-gui=qt4
```

Comment:
That's what Bitcoin is doing - i guess for compatibilty with 5 different Ubuntu versions...